src/pages/booking/EditBookingPage.vue

```diff
@@ -219,7 +219,7 @@ const isPartOfIt = computed(() => {
   }
   return false;
 });
-const busy = ref(false);
+const busy = ref(false); // what is this for?
```

Comment:
you set it busy so that you can reference in :loading prop of the button preventing user from re-clicking submit while the system is processing at backend. 
YOu may remove the comment.